cmd/operator/deploy/operator/05-deployment.yaml

```diff
@@ -56,6 +56,12 @@ spec:
           # Note this should match the --metrics-addr flag passed in to the operator args.
           # Default is 18080.
           containerPort: 18080
+        resources:
```

Comment:
Setting resource limits/requests here is great and nice.

Reply:
Cool!